saleor/plugins/webhook/tasks.py

```diff
@@ -136,6 +136,7 @@ def send_webhook_using_http(
     }
 
     response = requests.post(target_url, data=message, headers=headers, timeout=timeout)
+    response.raise_for_status()
```

Comment:
But by adding `raise_for_status` we lose failed request content and request headers otherwise returned in `WebhookResponse`. Nevertheless, we want to store them because they can contain valuable error information. Also, we are not treating all responses that don't raise an exception as successful because `WebhookResponse.status` depends on response status_code.